app/routes/_gcn/header/Header.tsx

```diff
@@ -135,6 +135,14 @@ export function Header() {
               >
                 Documentation
               </NavLink>,
+              <NavLink
+                className="usa-nav__link"
+                to="/news"
+                key="/news"
+                onClick={hideMobileNav}
+              >
+                News
+              </NavLink>,
```

Comment:
I think that I omitted News from the nav bar because it was too wide. What does this look like on desktop?